FogCreek/Glitch-Community
# Melted scorpion: toBlob support for edge

## Links
* https://melted-scorpion.glitch.me/
* https://glitch.manuscript.com/f/cases/3327868/Edge-does-not-support-toBlob-and-so-throws-errors
* https://sentry.io/organizations/fog-creek-software/issues/?project=1246508&query=is%3Aunresolved+toblob&statsPeriod=14d

## Changes:
* adds a toblob polyfill. Looks like we're currently throwing an error about not having toblob which we both log in sentry and then throw a generic error notification to our users about a problem uploading (even though it does seem to upload) Adding this polyfill fixes the issue

## How To Test:
* use browserstack (or a windows machine) and open edge, try to upload a file. In prod it should throw a red notification error, in this remix it does not.
